### PR TITLE
Add new environment type to constants

### PR DESCRIPTION
**Pull request by capmayer**
Add new enum constant for walls without enviroment in pipe enviroment heat transfer coefficient model type.

### CODE REVIEW

**Comment by williamjamir** [comment]
Please update the `HISTORY.rst` file and re-rerun the test with `force-regen` to update the expected values ;)

**Comment by nicoddemus** [comment]
>  re-rerun the test with force-regen to update the expected values

That would be `pytest --forge-regen`

File: HISTORY.rst [hunk 1]
```diff
@@ -11,6 +11,7 @@ History
 * Add new category: ``gas standard volume per time``, with same units as ``standard volume per time``.
 * Drop ``B_parameter`` as Lee-Chien method for surface tension is not supported anymore.
 * Add option to set the category for ``SecondaryVariable`` object
+* Add ``WallsWithoutEnvironment`` to PipeEnvironmentHeatTransferCoefficientModelType enum.
```
Comment (nicoddemus) [inline HISTORY.rst:14]:
```suggestion
* Add ``WallsWithoutEnvironment`` to ``PipeEnvironmentHeatTransferCoefficientModelType`` enum.
```

Comment (capmayer) [reply HISTORY.rst]: Ops, fixed it

**Comment by nicoddemus** [comment]
(Please squash all commits before merging 👍 )

**Review by nicoddemus — APPROVED** [review]

**Pull request merged**